security/pkg/probe/controller.go

```diff
@@ -35,6 +35,8 @@ const (
 	// LivenessProbeClientIdentity is the default identity for the liveness probe check
 	LivenessProbeClientIdentity   = "k8s.cluster.local"
 	probeCheckRequestedTTLMinutes = 60
+	// logEveryNChecks specifies we log once in every N successful checks.
+	logEveryNChecks = 20
```

Comment:
I mean why we choose 20, but not 100. Or how does this translate to the frequency of the log we see. Not a big deal, but would be easier to justify the value.

Reply:
This is really a subjective decision... It seems 100 is a more widely adopted number. I changed it to 100.

Reply:
I think @diemtvu 's point we can do a multiplication, 100 or 20 * internal of the probe to see how much log we generated.